src/guiFormSpecMenu.cpp

```diff
@@ -342,6 +342,20 @@ void GUIFormSpecMenu::parseSize(parserData* data,std::string element)
 	errorstream<< "Invalid size element (" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseListPredict(parserData* data, std::string element)
+{
+	if (!m_gamedef) {
+		errorstream<<"WARNING: invalid use of 'list_predict' with m_gamedef==0"<<std::endl;
+		return;
+	}
+	std::vector<std::string> parts = split(element, ',');
+	if (parts.size() == 2) {
```

Comment:
If you do a check for exactly 2 there's gonna be an error if someone adds additional parameters later. We already had this issue. For what I remember other (fixed) elements check for >=2 in case of formspec version of client not beeing same as formspec version provided by server.